snowpack/src/commands/dev.ts

```diff
@@ -814,6 +817,9 @@ export async function startServer(
     // Otherwise, load the file and respond if successful.
     try {
       const result = await loadUrl(reqUrl, {allowStale: true, encoding: null});
+      if (!result) {
+        throw NotFoundError;
+      }
```

Comment:
Here, in the dev server, we provide the same behavior: throw a 404 error if something doesn’t exist…